openapi/v2/core-command.yaml

```diff
@@ -44,13 +44,25 @@ components:
         message:
           description: "A field that can contain a free-form message, such as an error message."
           type: string
+    DeviceCoreCommand:
+      type: object
+      properties:
+        deviceName:
+          type: string
+        profileName:
+          type: string
+        coreCommands:
+          type: array
+          items:
+            $ref: '#/components/schemas/CoreCommand'
+      required:
+        - deviceName
+        - profileName
```

Comment:
Don't need the required section on Response only objects. i.e no validate performed as we build the response DTO.

Reply:
Agree.  I've removed required section out of response DTO.  Please check https://github.com/edgexfoundry/edgex-go/pull/3153/commits/47b4c9e6fc4044b20e859d8e973f792df8961e2f